docs/application/dotnet/guides/account/app-history.md

```diff
@@ -84,24 +84,24 @@ To retrieve application usage statistics for a given time period, and check deta
 
 To retrieve battery usage statistics for a given time period, and check detailed statistics information, such as the battery consumption of the used applications:
 
-1.  To retrieve the battery consumption per application, create an instance of the [Tizen.Context.AppHistory.BatteryStatistics](https://samsung.github.io/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatistics.html) class:
+1.  To retrieve the battery consumption per application, create an instance of the [Tizen.Context.AppHistory.BatteryStatistics](/application/dotnet/api/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatistics.html) class:
 
-    ```
+    ```csharp
     BatteryStatistics batteryConsumedApp = new BatteryStatistics(BatteryStatistics.SortOrderType.ConsumptionMost);
     ```
 
 2.  To get the information about the application battery consumption, use the `Query()` method of the `Tizen.Context.AppHistory.BatteryStatistics` class.
 
     For example, to retrieve battery consumption history since the device was last fully charged, use a `DateTime` instance returned by the `GetLastFullyChargedTime()` method as the `startTime` parameter of the `Query()` method:
 
-    ```
+    ```csharp
     DateTime time = BatteryStatistics.GetLastFullyChargedTime();
     IReadOnlyList<BatteryStatisticsData> batteryConsumedAppList = batteryConsumedApp.Query(time, DateTime.Now, 5);
     ```
 
-3.  The `Query()` method returns a sorted list of [Tizen.Context.AppHistory.BatteryStatisticsData](https://samsung.github.io/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatisticsData.html) class instances. To enumerate the list:
+3.  The `Query()` method returns a sorted list of [Tizen.Context.AppHistory.BatteryStatisticsData](/application/dotnet/api/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatisticsData.html) class instances. To enumerate the list:
 
-    ```
+    ```csharp
```

Comment:
- Dependencies

Reply:
Could you please clarify what should be changed?